### PR TITLE
update for lambda metrics extension

### PR DESCRIPTION
**Pull request by kubawach**
- added extension description
- added links to docs for each supported layer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
-<h2>Overview</h2>
+<h1>Splunk APM lambda layers</h1>
```
Comment (breedx-splk) [inline README.md:1]: Not terribly important, but maybe next time we can favor markdown `#` and `##` etc for making headers, instead of embedding html?

**Review by breedx-splk — COMMENTED** [review]
This looks good to me. 

Can we clarify where the examples have moved to or why they've been removed? Thanks!

File: README.md (hunk 1 above)
Comment (kubawach) [reply README.md:1]: That's a good idea. I don't even know why it's HTML in the first place ;)

**Comment by kubawach** [comment]
@breedx-splk wrt examples - each project keeps them within main repo. For extension it was a copy made by build, but I decided to remove this duplication so that everything is aligned. Hope that this makes sense not only to me 🤷  :)

**Pull request merged**